docs/blog/ibis-version-4.0.0-release.md

```diff
@@ -0,0 +1,38 @@
+# Ibis v4.0.0
+
+**by Patrick Clarke**
+
+09 January 2023
+
+## Introduction
+
+Ibis 4.0 has officially been released as the latest version of the package.
+With this release comes of more backends, improved functionality, several refactors, bug fixes, and more.
+A full list of the changes can be found in the project release notes [here](../release_notes.md).
+Let’s talk about some of the new changes 4.0 brings for Ibis users.
+
+## Backends
+
+Ibis 4.0 brings [Polars](https://ibis-project.org/docs/latest/backends/Polars/), [Snowflake](https://ibis-project.org/docs/dev/backends/Snowflake/), and [Trino](https://ibis-project.org/docs/dev/backends/Trino/) into and already-impressive stock of supported backends.
+The [Polars](https://www.pola.rs/) backend adds another way for users to work locally with DataFrames, and the [Snowflake](https://www.snowflake.com/en/) and [Trino](https://trino.io/) backends adds a free and familiar python API to popular data warehouses.
+
+## Functionality
+
+There are a lot of improvements incoming, but some notable changes include:
+
+- [read](https://github.com/ibis-project/ibis/pull/4759) API ([including explicit functions](https://github.com/ibis-project/ibis/pull/5005)): allows users to read various file formats directly into their [configured `default_backend`](https://ibis-project.org/docs/dev/api/config/?h=default#ibis.config.Options) (default DuckDB).
+- [to_pyarrow](https://github.com/ibis-project/ibis/pull/4454#issuecomment-1262640204): users can now return PyArrow objects (Tables, Arrays, Scalars) and therefore grants all of the functionality that PyArrow provides
```

Comment:
May also want to include `to_pyarrow_batches`, which returns a pyarrow `RecordBatchReader` for streaming result sets.

Reply:
[updated](https://github.com/ibis-project/ibis/pull/5177/files#diff-2ec843e97605581ceada30571b1090aabe0bf588427981f61eceb5086dfb1c9bR27)